### PR TITLE
[vagrant] implemented 2 independent vagrant vms

### PR DESCRIPTION
**Pull request by M0ses**
* one for developing the frontend
** fixed output of provisioner script (RAILS_ENV=development was missing)
* one for testing deploment and appliance
* added code to make Vagrantfile more usable with vagrant-libvirt

### CODE REVIEW

**Comment by M0ses** [comment]
!!!
This commit has not been tested with virtualbox as provider (only libvirt).
!!!
 Maybe somebody with an running Virtualbox could help ?

**Comment by ChrisBr** [comment]
tested with virtualbox! Works as expected for me!

LGTM :+1:

File: bootstrap.sh [hunk 1]
```diff
@@ -43,22 +52,15 @@ else
...
-  echo -e "\n\nWARNING: You have already configured your app in config/options.yml." 
-  echo -e "WARNING: Please make sure this configuration works in this vagrant box!\n\n" 
-fi 
 
 echo "Setting up your OBS test backend..."
 # Put the backend data dir outside the shared folder so it can use hardlinks
 # which isn't possible with VirtualBox shared folders...
 mkdir /tmp/vagrant_tmp
 chown vagrant:users /tmp/vagrant_tmp
+chown vagrant:users -R /vagrant/src/api/log/*
```
Comment (hennevogel) [inline bootstrap.sh]: Why?

Comment (M0ses) [inline bootstrap.sh]: Sorry, took me some time to understand as I`m not using virtualbox (I use libvirt) and vagrant-libvirt does not use share folders, its rsyncing to the box and files owned by root afterwards. 

I will modify this to use an "bind" filesystem like you do with tmp. Would this be a solution for you ?

**Pull request merged**